docs/pages/changelog/index.js

```diff
@@ -10,6 +10,10 @@ const unreleased = [
     version: 'Unreleased',
     date: '?',
     children: html`<ul>
+      <li>
+        <p><code>vl-breadcrumb</code></p>
+        <p>When migrating the breadcrumb, we fixed a bug where new breadcrumb items did not get the correct styling.</p>
```

Comment:
Hiermee is niet echt duidelijk wat er juist is opgelost, de items kregen wel de juiste styling onload maar als er daarna nog iets veranderde ging het mis.

`When migrating the breadcrumb, we mentioned that the right styles where only applied on load. When the components children changed, the styled did not change correspondingly.` Of zo iets.

Ook mag de 'explosion' emoji naast het titeltje, om aan te geven dat de changes breaking zijn tov de legacy component. Zoals bv lijn 120 vd changelog file.